docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md

```diff
@@ -267,6 +282,28 @@ Master is missing from Raft group or has follower lag higher than `$threshold` s
 max by (node_prefix) (count by (node_prefix, exported_instance) (max_over_time(yb_node_underreplicated_tablet{node_prefix="$node_prefix"}[5m])) > 0)
 ```
 
+#### Tablet server average read latency is high
+
+Average read latency of tablet server for universe `'$universe_name'` is above `$threshold%` ms. The current value is `$value` milliseconds.
+
+```expression
+(avg by (universe_uuid) (rate(rpc_latency_sum{export_type="tserver_export", service_type="TabletServerService", universe_uuid="__universeUuid__", server_type="yb_tserver", service_method="Read"}[5m])))
+       / (avg by (universe_uuid)
+(rate(rpc_latency_count{export_type="tserver_export", service_type="TabletServerService", universe_uuid="__universeUuid__", server_type="yb_tserver", service_method="Read"}[5m])) * 1000)
+      {{ query_condition }} {{ query_threshold }}
+```
+
+#### Tablet server average write latency is high
+
+Average write latency of tablet server for universe '{{ $labels.source_name }}' `'$universe_name'` is above `$threshold%` ms. The current value is `$value` milliseconds.
```

Comment:
labels.source_name  - backticks? and this variable isn't in read latency above - is that right?

Reply:
Yeah I think the variable `labels.source_name` is not supposed to be here?